api/src/main/java/gov/cms/ab2d/api/controller/BulkDataAccessAPI.java

```diff
@@ -321,7 +318,7 @@ private String getUrlPath(Job job, String filePath) {
     )
     @ResponseStatus(value = HttpStatus.OK)
     @GetMapping(value = "/Job/{jobUuid}/file/{filename}")
-    public ResponseEntity<Void> downloadFile(
+    public ResponseEntity<String> downloadFile(
```

Comment:
Usually changing return types affects Swagger documentation, because Spring Fox generates Swagger from the code. Please take a look at Swagger to make sure nothing odd popped up as a result of this. @lailasharshar might help out as well.